src/modules/Pagination/Pagination.scss

```diff
@@ -0,0 +1,63 @@
+@import 'index.scss';
+
+.j-pagination {
+  .Pagimagic-nav {
+    &-item {
+    	display: inline-block;
+      vertical-align: top;
+      height: 30px;
+      line-height: 30px;
+      background-color: $hf-color-white;
+	    border: $hf-border;
+	    font-size: 12px;
+	    text-align: center;
+  	  color: $hf-color-gray-medium;
+      padding: 0 8px;
+      box-sizing: border-box;
+      transition: background-color 0.05s, color 0.05s, border-color 0.05s;
+      cursor: pointer;
+
+      &.active {
+        background-color: $hellofresh-color;
+        border-color: $hellofresh-color;
+        color: $hf-color-white;
+      }
+
+      &--prev,
+      &--next {
+        position: relative;
+        width: 20px;
+        height: 30px;
+        border: none;
+        background: none;
+
+        &:before {
+          content: '';
+          display: block;
+          position: absolute;
+          width: 8px;
```

Comment:
`content` it's for implementing `:before` pseudo. And other yes, for aligning. But what could be the worst scenario? 
Of course it's possible to not think about that by using `left: 50%; top: 50%; transform: translate(-50%, -50%)` which is more powerful, but much heavier for browser, and sometimes can make some blurry effect. So, when it's possible to know the exact size of the icon, I prefer to use more exact sizes.